### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by pratap-panabaka**
This project will lay foundations for the Bookstore website. Create the React part of the app. Also, set up routing using React Router.

### CODE REVIEW

File: src/components/Categories.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import Header from './Header';
+
+const Categories = () => (
+  <>
+    <Header />
```
Comment (tufoinnkuo10) [inline src/components/Categories.js:6]: - [ ] great job so far.. Everything is perfect. Kindly ensure the categories page should display  a button with text "Check status"

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @PRATAP-KUMAR 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally

Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: src/components/Categories.js (hunk 1 above)
Comment (PRATAP-KUMAR) [reply src/components/Categories.js:5]: @tufoinnkuo10 

Hi Thank you for your review, the document says

> Should have a form for adding a book (no functionality yet).
> Create a separate component for this form (with inputs for a title and an author [IMPORTANT in the design you can see an input for a category - please replace it with an author.]).
> Categories:
> Should display a button "Check status" only.
> Styling is not required at this point.

https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/project_init_project_wth_comps_v2_0.md

Waiting for your valuable reply

Thank You

Comment (pratap-panabaka) [reply src/components/Categories.js:6]: @tufoinnkuo10 

Thank you for your review, added it now.

image

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**